src/unix/bsd/mod.rs

```diff
@@ -10,6 +10,11 @@ pub type sa_family_t = u8;
 pub type pthread_t = ::uintptr_t;
 pub type nfds_t = ::c_uint;
 
+#[cfg(not(target_os = "dragonfly"))]
+macro_rules! utsname_len { () => (256) }
+#[cfg(target_os = "dragonfly")]
+macro_rules! utsname_len { () => (32) }
```

Comment:
I tried using a const first, but then ```libc-test``` took the const name and tried to match it up.